code/validation/test-formatting.py

```diff
@@ -48,7 +48,12 @@ def filename_match_forecast_date(filename):
             return "ERROR %s forecast filename date %s does match forecast_date column %s" % (
                 filename, file_forecast_date, forecast_date_column)
         else:
-            return None
+            today = datetime.date.today()
+            if (forecast_date_column != (str(today))):
```

Comment:
I think in the US they were going to allow either `today` or `today + 1` (not sure why - I've asked)

Reply:
Updated after decision to permit only the Monday submission date in either forecast_date or filename.